src/styles/theme.scss

```diff
@@ -0,0 +1,397 @@
+@font-face {
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: 600;
+  src: url('./fonts/Poppins-Medium.ttf') format('truetype'),
+    url('./fonts/poppins-medium-webfont.woff') format('woff'),
+    url('./fonts/poppins-medium-webfont.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: bold;
+  src: url('./fonts/Poppins-SemiBold.ttf') format('truetype'),
+    url('./fonts/poppins-semibold-webfont.woff') format('woff'),
+    url('./fonts/poppins-semibold-webfont.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: normal;
+  src: url('./fonts/Poppins-Regular.ttf') format('truetype'),
+    url('./fonts/poppins-regular-webfont.woff2') format('woff2'),
+    url('./fonts/poppins-regular-webfont.woff') format('woff');
+}
+
+/* ------------------------------
+
+   Variables
+
+------------------------------ */
+
+// Contemplating Action variables
+$purple: #662483;
+$purple-light: #8b1abc;
+$blue-darkest: #1a1e64;
+$blue-dark: #30336d;
+$blue-light: #4575ba;
+$black: #3c3c3b;
+$white: #f5f5f5;
+$grey-light: #eee;
+$grey-light-purple: #e4e1e7;
+$lime-green: #07c103;
+
+// Utility variables
+$white-pure: #ffffff;
+$black-pure: #000000;
+$shadow: rgba(89, 97, 116, 0.4);
+
+// Theme colours
+$primary-colour: $blue-darkest;
+$secondary-colour: $blue-light;
+$text-colour: $black;
+$interaction-colour: $purple;
+$focus-colour: $lime-green;
+
+/* ------------------------------
+
+   Defaults
+
+------------------------------ */
+
+*:focus {
+  outline-color: $focus-colour;
+}
+
+:root {
+  font-size: 100%;
+  font-family: 'Poppins', 'Arial';
+  line-height: 1.5;
+}
+
+body {
```

Comment:
I'm wondering if it'll be worth moving some stuff into a 3rd scss - `brand.scss` and keep much of this theme stuff that 'should' remain the same for all 4 sites... to lessen the probability of a brand breaking something.